modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -7,34 +7,64 @@
 
 Uninstalling the web terminal is a two-step process:
 
-. Delete the components and custom resources (CRs) that were added when you installed the Operator.
-. Uninstall the Web Terminal Operator.
+. Uninstall the Web Terminal Operator and related custom resources (CRs) that were added when you installed the Operator.
+. Uninstall the DevWorkspace Operator and its related custom resources that were added as a dependency of the Web Terminal Operator.
 
 Uninstalling the Web Terminal Operator does not remove any of its custom resource definitions (CRDs) or managed resources that are created when the Operator is installed. These components must be manually uninstalled for security purposes. Removing these components also allows you to save cluster resources by ensuring that terminals do not idle when the Operator is uninstalled.
 
 .Prerequisites
 * Access to an {product-title} cluster using an account with `cluster-admin` permissions.
 
-== Deleting the web terminal components and custom resources
+== Removing the Web Terminal Operator and the custom resources that support it
 
-Use the CLI to delete the CRs that are created during installation of the  Web Terminal Operator.
+Use the CLI to delete any existing web terminals and CRs that were created during the installation of the Web Terminal Operator.
```

Comment:
Since the procedure uses the console as well as the CLI, maybe we can modify this sentence to say 'Use the console and the CLI to delete...'